### DISCUSSION TITLE
something wrong when running Freesufer.sh on example

### DISCUSSION THREAD

**Discussion by LordXX505** [Washington-University/HCPpipelines#237] [Q&A] [answered]
I follow steps described on readme but 
I get an error when I do   ./FreesurferPipelineBatch.sh
It log says,"recon-all exit  1"
but I singlely test freesurfer,it works well that i use the exmaple data given from fslwiki
Im so puzzled

**Comment by coalsont** [comment]
Did you run PreFreesurfer first?  Please post the entire log (stderr and stdout) so we can see any other messages that came before that.

**Reply by LordXX505** [reply]
I re run the PreFreesurfer  and then run the Freesurfer.sh ,same problem occured and here is the log

\(base) hwx@hwx-MS-7D24:\~/newdisk/projects/Examples/Scripts$ ./PreFreeSurferPipelineBatch.sh 
StudyFolder: /home/hwx/newdisk/projects/hcp_examples
Subjlist: 100307
EnvironmentScript: /home/hwx/newdisk/projects/Examples/Scripts/SetUpHCPPipeline.sh
Run locally: 
100307
Found 1 T1w Images for subject 100307
Found 1 T2w Images for subject 100307
About to use fsl_sub to queue or run /home/hwx/newdisk/projects/PreFreeSurfer/PreFreeSurferPipeline.sh
24271
(base) hwx@hwx-MS-7D24:~/newdisk/projects/Examples/Scripts$ ./FreeSurferPipelineBatch.sh 

100307
About to use fsl_sub to queue or run /home/hwx/newdisk/projects/FreeSurfer/FreeSurferPipeline.sh
/home/hwx/newdisk/projects/FreeSurfer/FreeSurferPipeline.sh
Error submitting job - Thu Oct  6 22:32:22 CST 2022:FreeSurferPipeline.sh: While running '/home/hwx/newdisk/projects/FreeSurfer/FreeSurferPipeline.sh --subject=100307 --subjectDIR=/home/hwx/newdisk/projects/hcp_examples/100307/T1w --t1=/home/hwx/newdisk/projects/hcp_examples/100307/T1w/T1w_acpc_dc_restore.nii.gz --t1brain=/home/hwx/newdisk/projects/hcp_examples/100307/T1w/T1w_acpc_dc_restore_brain.nii.gz --t2=/home/hwx/newdisk/projects/hcp_examples/100307/T1w/T2w_acpc_dc_restore.nii.gz':
Thu Oct  6 22:32:22 CST 2022:FreeSurferPipeline.sh: ERROR: recon-all command failed with return_code: 1

set -- --subject=100307       --subjectDIR=/home/hwx/newdisk/projects/hcp_examples/100307/T1w       --t1=/home/hwx/newdisk/projects/hcp_examples/100307/T1w/T1w_acpc_dc_restore.nii.gz       --t1brain=/home/hwx/newdisk/projects/hcp_examples/100307/T1w/T1w_acpc_dc_restore_brain.nii.gz       --t2=/home/hwx/newdisk/projects/hcp_examples/100307/T1w/T2w_acpc_dc_restore.nii.gz
. /home/hwx/newdisk/projects/Examples/Scripts/SetUpHCPPipeline.sh
(base) hwx@hwx-MS-7D24:~/newdisk/projects/Examples/Scripts$

**Comment by glasserm** [comment, marked as answer]
Also such questions are best asked on the hcp-users mailing list.

**Reply by LordXX505** [reply]
thks,i will try this

**Comment by glasserm** [comment]
What is the error in the recon-all logfile?

**Comment by betsy1226** [comment]
Have you solved the problem? I met the same error and felt puzzled.
